lib/fauna/query.rb

```diff
@@ -820,6 +820,30 @@ def not_(boolean)
       Expr.new not: Expr.wrap(boolean)
     end
 
+    ##
+    # Converts an expression to a string literal.
```

Comment:
We should add the link to their section in the docs, if there is a link.

Reply:
There isn't one yet.